crates/net/network/src/transactions.rs

```diff
@@ -778,7 +773,7 @@ where
 
         // Advance all requests.
         while let Poll::Ready(Some(GetPooledTxResponse { peer_id, result })) =
-            this.inflight_requests.poll_next_unpin(cx)
+            this.transaction_fetcher.inflight_requests.poll_next_unpin(cx)
```

Comment:
let's add a `fn poll` to the fetcher and call this instead,

because eventually we need to handle more things in the fetcher, for example, keep track of inflight blobs etc.
modifying this will be easier when we already have `Fetcher::poll`

Reply:
i wasn't sure what to put in `fn poll` so i just made it the "minimum viable" functionality for now